charts/argo-templates/templates/diff-bq-table.yaml

```
@@ -137,7 +137,9 @@ spec:
           - name: rows-to-append-count
             valueFrom:
               parameter: '{{ "{{tasks.query-row-count.outputs.result}}" }}'
-
+          - name: join-table-name-output
```

The `-output` suffix is redundant, would just do:
```suggestion
          - name: join-table-name
```